src/components/new-request/name-input.vue

```diff
@@ -136,9 +136,9 @@ export default class NameInput extends Vue {
       if (this.searchValue) {
         // hide modal and perform name analysis
         this.setMrasSearchInfoModalVisible(false)
-        if (this.getIsXproFlow) this.$root.$emit('showSpinner', true)
-        if (this.searchValue) await this.startAnalyzeName(null)
-        if (this.getIsXproFlow) this.$root.$emit('showSpinner', false)
```

Comment:
Since we are going to analyze the name regardless, there is no need to show the spinner only for xpro flow.

Also, there's no need for the "if" on line 140 as we already checked this on line 136.